ChangeFeedLabSolution/ChangeFeedProcessor.cs

```diff
@@ -10,65 +10,110 @@
 /// </summary>
 namespace ChangeFeedFunction
 {
+    using System;
     using System.Collections.Generic;
-    using System.Configuration;
-    using System.Text;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+    using Azure.Messaging.EventHubs;
+    using Azure.Messaging.EventHubs.Producer;
     using Microsoft.Azure.Documents;
-    using Microsoft.Azure.EventHubs;
     using Microsoft.Azure.WebJobs;
-    using Microsoft.Azure.WebJobs.Host;
-    using Newtonsoft.Json;
+    using Microsoft.Extensions.Logging;
 
     /// <summary>
     /// Processes events using Cosmos DB Change Feed.
     /// </summary>
-    public static class ChangeFeedProcessor
+    public class ChangeFeedProcessor
     {
         /// <summary>

```

Comment:
This comment does not match the member that it is associated with.

Reply:
@jsquire - Removed.